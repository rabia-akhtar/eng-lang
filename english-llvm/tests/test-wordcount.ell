int main()
{
  string essay = "This is a test to see how many words are in this essay. There are two sentences.";
  int count = word_count(essay);
  print_string(essay);
  print(count);
  
}
