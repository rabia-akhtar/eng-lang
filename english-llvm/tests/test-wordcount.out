This is a test to see how many words are in this essay. There are two sentences.
17
